analyzers/tests/SonarAnalyzer.UnitTest/TestCases/StaticFieldVisible.cs

```diff
@@ -6,9 +6,23 @@ public class StaticFieldVisible
     {
         public static double Pi = 3.14;  // Noncompliant
```

Comment:
Here I would also add a couple of compliant cases with 0 and 1 modifiers, to hit the `node.Modifiers.Count > 1` branch:
```cs
int Field1 = 3;
[SomeAttribute] int Field2 = 3;
static int Field3 = 3; // Compliant, static but private
volatile int Field4 = 3; // Compliant, because non-static
```

Reply:
We already have 100% coverage  -
`int Field1 = 3;` - Added
`[SomeAttribute] int Field2 = 3;` - Added
` static int Field3 = 3; // Compliant, static but private` - It's already there
`volatile int Field4 = 3; // Compliant, because non-static` - IMO this adds more noise in this case as we already have non static examples.

Reply:
The coverage metrics exposed by AltCover only seems to support line and branch coverage. It doesn't seem to support (full) condition coverage (where you cover all possible values of each boolean expression of a condition, with the exception of short-circuit evaluations).

Given the previous ordering of conditions in the `if` and given `A = node.Modifiers.Count > 1`, `B = node.Modifiers.Any(SyntaxKind.StaticKeyword)`, `C =  !node.Modifiers.Any(SyntaxKind.PrivateKeyword)` and `D = !node.Modifiers.Any(SyntaxKind.ReadOnlyKeyword);` what I wanted to cover was:
- `int Field` => `!B`
- `static int Field3 = 3` => `B && !A` => was indeed already there
- ...
- `static volatile int Field` => `B && A && C && D`

Given the new ordering of conditions and the test cases you added, you should have good condition coverage now.